ui/de-lib/src/main/java/org/iplantc/de/admin/desktop/client/toolAdmin/view/dialogs/ToolAdminDetailsDialog.java

```diff
@@ -0,0 +1,191 @@
+package org.iplantc.de.admin.desktop.client.toolAdmin.view.dialogs;
+
+import org.iplantc.de.admin.desktop.client.toolAdmin.ToolAdminView;
+import org.iplantc.de.admin.desktop.client.toolAdmin.events.DeleteToolSelectedEvent;
+import org.iplantc.de.admin.desktop.client.toolAdmin.events.SaveToolSelectedEvent;
+import org.iplantc.de.admin.desktop.client.toolAdmin.view.ToolAdminDetailsView;
+import org.iplantc.de.client.models.IsHideable;
+import org.iplantc.de.client.models.tool.Tool;
+import org.iplantc.de.client.models.tool.ToolAutoBeanFactory;
+import org.iplantc.de.client.models.tool.ToolContainer;
+import org.iplantc.de.client.models.tool.ToolDevice;
+import org.iplantc.de.client.models.tool.ToolImplementation;
+import org.iplantc.de.client.models.tool.ToolTestData;
+import org.iplantc.de.client.models.tool.ToolVolume;
+import org.iplantc.de.client.models.tool.ToolVolumesFrom;
+import org.iplantc.de.commons.client.views.dialogs.IPlantDialog;
+
+import com.google.common.collect.Lists;
+import com.google.gwt.event.shared.HandlerRegistration;
+import com.google.gwt.event.shared.HasHandlers;
+import com.google.gwt.user.client.ui.HTML;
+import com.google.inject.Inject;
+
+import com.sencha.gxt.core.client.dom.ScrollSupport;
+import com.sencha.gxt.widget.core.client.box.AlertMessageBox;
+import com.sencha.gxt.widget.core.client.box.ConfirmMessageBox;
+import com.sencha.gxt.widget.core.client.button.TextButton;
+import com.sencha.gxt.widget.core.client.container.FlowLayoutContainer;
+import com.sencha.gxt.widget.core.client.event.DialogHideEvent;
+import com.sencha.gxt.widget.core.client.event.SelectEvent;
+
+/**
+ * @author jstroot
+ * @author aramsey
+ */
+public class ToolAdminDetailsDialog extends IPlantDialog implements IsHideable,
+                                                                    SaveToolSelectedEvent.HasSaveToolSelectedEventHandlers,
+                                                                    DeleteToolSelectedEvent.HasDeleteToolSelectedEventHandlers {
+
+    private final ToolAdminDetailsView view;
+    private final ToolAdminView.ToolAdminViewAppearance appearance;
+    @Inject
+    ToolAutoBeanFactory factory;
+
+    @Inject
+    public ToolAdminDetailsDialog(final ToolAdminDetailsView view,
+                                  final ToolAdminView.ToolAdminViewAppearance appearance) {
+        super(true);
+        this.view = view;
+        this.appearance = appearance;
+
+        setHideOnButtonClick(false);
+        setHeadingText(appearance.dialogWindowName());
+        setResizable(true);
+        setPixelSize(1000, 500);
+        setMinHeight(200);
+        setMinWidth(500);
+
+        addHelp(new HTML(appearance.toolAdminHelp()));
+
+        setPredefinedButtons(PredefinedButton.OK, PredefinedButton.CANCEL, PredefinedButton.CLOSE);
+        getOkButton().setText(appearance.dialogWindowUpdateBtnText());
+        getDeleteButton().setText(appearance.dialogWindowDeleteBtnText());
```

Comment:
The `Delete` button should probably go next to the `Add` button in the main `Tool Admin` toolbar, rather than in the edit dialog. This would also be more consistent with the other tabs like `System Messages` and `Metadata Templates`.